lib/ansible/modules/network/nxos/nxos_vrf.py

```diff
@@ -318,6 +322,14 @@ def map_obj_to_commands(updates, module):
                             commands.append('interface {0}'.format(i))
                             commands.append('no switchport')
                             commands.append('no vrf member {0}'.format(name))
+                elif interfaces and interfaces[0] == 'default':
+                    if obj_in_have['interfaces']:
```

Comment:
what if there is no `obj_in_have['interfaces']`?

Reply:
Basically, if interfaces is 'default' and the current object has some interfaces, this code removes them by doing 'no' cmd on each of them. There is no need for 'else' here because if the current object has no interfaces and user wants 'default', we are already in that state so no new commands need to be generated and impotence is maintained.